### PR TITLE
feat: on new-opened models, enable all projects

### PR DESCRIPTION
**Pull request by Skn0tt**
Closes https://github.com/microsoft/playwright/issues/33865.

### CODE REVIEW

File: src/testModel.ts [hunk 1]
```diff
@@ -715,11 +713,13 @@ export class TestModelCollection extends DisposableBase {
...
     model.isEnabled = enabled;
-    if (userGesture)
-      this._saveSettings();
     model.reset();
     const configSettings = this._configSettings(model.config);
-    model._loadModelIfNeeded(configSettings).then(() => this._didUpdate.fire());
+    model._loadModelIfNeeded(configSettings).then(() => {
+      this._didUpdate.fire();
+      if (userGesture)
+        this._saveSettings();
```
Comment (Skn0tt) [inline src/testModel.ts:721]: Before, we were saving settings immediately before reading them in `this._configSettings`. This breaks the "no settings yet" path in `_loadModelIfNeeded`, and broke tests in`track-config.spec.ts`. By moving the save settings call after the `_loadModel`, we keep the "no settings yet" path working.

Comment (pavelfeldman) [reply src/testModel.ts:720]: How is the following case handled:

- I am like 90% of our users
- I created a playwright project a year ago, I only have one config, 'chromium' was checked for me automatically
- For the last year I was pressing "Run" button and I was getting all my tests run against Chromium
- I don't expect this to change when I update extension.

Comment (Skn0tt) [reply src/testModel.ts:720]: Well when I wrote this I tried to change it a way that existing users don't have any changes. This is a complicated thing though, so let me try writing a test to verify this.

Comment (Skn0tt) [reply src/testModel.ts:720]: Oh no :/ If the user never made a gesture that invokes `_saveSettings`, then their workspace settings are undefined, making them indistinguishable from users opening a new workspace. Not sure why, but I somehow assumed that we'd save settings immediately on opening. I'll have to rethink this.

**Comment by Skn0tt** [comment]
Here's a PR to add that marker: https://github.com/microsoft/playwright-vscode/pull/577

After rolling that out, we'll wait for a couple of weeks for folks to update + activate that code in their Playwright workspace. Those workspaces will then be detectable as existing workspaces, allowing us to opt them out the new project defaults.

**Pull request closed without merging**